release-lock-file-spec.md

```diff
@@ -23,52 +152,69 @@ single JSON object.
 
 ## Filename
 
-When creating a release, convention is to name this document `<version>.json`.
+When creating a release, convention is to name this document using one of the following conventions.
 
-* The `1.0.0` Release would be named `1.0.0.json`
+* In cases where multiple *Release Lockfiles* are being store alongside one another, convention is to use `<version>.json`.  For example, the `1.0.0` release would be named `1.0.0.json`.  
+* In cases where a single *Release Lockfile* is being stored, convention is to use `lock.json`.
 
-Package managers may keep their own copy of this file as part of their package installation process.  In these cases it is up to the package manager to decide on a naming convention.  Package managers **should** keep the `.json` file extension.
+Package managers **should not** rely on these naming conventions nor are they
+required to follow them.
```

Comment:
Perhaps. But to use an example, digging inside `node_modules` or python eggs (or whatever they're called) is not something a user would do on a regular basis. The only time I've dug into package locations disk is when I was publishing my own package (in the case of python; I'd install the package globally and then see what got installed because it was unclear a priori); or I was dealing with some weird `npm`'ism in how `npm` breaks import statements in very rare cases.

This is all to say the innards of the `installed_contracts` directory isn't something that's user facing. If `yarn` and `npm` want to do it differently, for instance, that's up to them (yarn saves a `yarn.lock`; npm does not). I just don't think the spec needs to worry about this aspect of the implementation.